spikesorters/yass/yass.py

```diff
@@ -0,0 +1,280 @@
+import copy
+from pathlib import Path
+import os
+import numpy as np
+from numpy.lib.format import open_memmap
+import sys
+import yaml
```

Comment:
yaml is not mandatory for spikesorter you have to put in your try except with yass

Reply:
Done.